source/deploy-quickstart.md

```diff
@@ -6,6 +6,8 @@ description: Learn how to quickly deploy an app to Galaxy with these step-by-ste
 
 Galaxy makes it simple to deploy, scale, and monitor your Meteor app. This quickstart guide will walk new users through deploying your first app. It is the abridged version of the [comprehensive guide](/deploy-guide.html) aimed at developers who have experience deploying apps with remote databases.
 
+If you desire to deploy your app using your machine, you can continue reading this. But we have an easier and faster way of deploying your app, which is by using the feature [Push To Deploy](/push-to-deploy.html), and if this is your first time deploying a Meteor application, this is what we'd recommend you to use.
```

Comment:
The same comment, we should refactor this page to recommend push to deploy as the quickstart.